Mathlib/Analysis/Normed/Unbundled/AlgNormOfGalois.lean

```diff
@@ -0,0 +1,143 @@
+/-
+Copyright (c) 2023 María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: María Inés de Frutos-Fernández
+-/
+import Mathlib.Analysis.Normed.Unbundled.FiniteExtension
+import Mathlib.LinearAlgebra.FreeModule.Finite.Matrix
+
+/-!
+# algNorm_of_auto and algNorm_of_galois
+
+Let `K` be a nonarchimedean normed field and `L/K` be a finite algebraic extension. In the comments,
+`‖ ⬝ ‖` denotes any power-multiplicative algebra norm on `L` extending the norm  on `K`.
```

Comment:
```suggestion
`‖ ⬝ ‖` denotes any power-multiplicative algebra norm on `L` extending the norm on `K`.
```

Reply:
You call it both an "algebra norm on K" and an "algebra norm on L" in the docstring. Shouldn't it be called a "K-algebra norm on L" or something like that?

Reply:
The "algebra norm on K" was a typo. I will use "K-algebra norm on L".